server/conf/timely-standalone.yml

```diff
@@ -0,0 +1,47 @@
+timely:
+  ip: 0.0.0.0
+  port:
+    put: 54321
+    query: 54322
+    websocket: 54323
+  instance-name: TimelyStandalone
```

Comment:
Ok, how about this instead? I left it tcp-port in the server section since Dave mentioned he was planning to add a UDP port.

```yaml
timely:
  metrics-table: timely.metrics
  meta-table: timely.meta
  metric-age-off-days: 7
  metrics-report-ignored-tags:
  accumulo:
    instance-name:
    zookeepers:
    username:
    password:
    write:
      latency:
      threads:
      buffer-size:
    scan:
      threads:
  server:
    ip:
    tcp-port:
  security:
    allow-anonymous-access: false
    session-max-age: 86400
    ssl:
      certificate-file:
      key-file:
      key-password:
      use-generated-keypair: false
      trust-store-file:
      use-openssl: true
      #use-ciphers:
  http:
    ip:
    port:
    host:
    redirect-path: /secure-me
    strict-transport-max-age: 608400
    cors:
      allow-any-origin: false
      allow-null-origin: false
      allow-credentials: true
      allowed-origins:
      allowed-methods: DELETE,GET,HEAD,OPTIONS,PUT,POST
      allowed-headers: content-type
  websocket:
    ip:
    port:
    timeout: 60
    subscription-lag: 120
  meta-cache:
    expiration-minutes: 60
    initial-capacity: 2000
    max-capacity: 10000
  visibility-cache:
    expiration-minutes: 60
    initial-capacity: 2000
    max-capacity: 10000
```